app/dashboard/templates/profiles/organization.html

```diff
@@ -25,15 +25,17 @@
         <li><b>{{ count_bounties_on_repo }}</b> {% trans "bounties" %}</li>
         <li><b>{{ sum_eth_on_repos|floatformat:2 }}</b> {% trans "ETH" %}</li>
         <li class="works_with_list">
-          {% if works_with_org|length != 0 %}
+          {% if works_with_org|length %}
             <span class="font-body"><i class="fa fa-code"></i> {% trans "Contributors" %}:</span>
-            {% for profile, num_times in works_with_org.items|slice:":5" %}
-              <a href="{% url 'profile' profile %} ">
-                <img src='{% avatar_url profile %}' title="<div class='tooltip-info tooltip-xs'>{{ profile }}: {{ num_times }} times</div>">
-              </a>
+            {% for profile, num_times in works_with_org.items %}
```

Comment:
but what if you work with lots of ppl, like gitcoin.co/profile/owocki does!!?

Reply:
@octavioamu / @owocki  Assuming the profile has worked with 15 folks 
would it make sense to 
- keep worked with card at a fixed height
- show 8 profiles you've worked with  `+ 7 More`
- On clicking  `+ 7 More` show all the profiles but make it scrollable

Reply:
That is right @thelostone-mc the popover show everyone